nginx.conf

```diff
@@ -2,6 +2,11 @@ server {
     listen       80;
     listen  [::]:80;
     server_name  localhost;
+    gzip on;
+    gzip_types text/plain application/xml text/css application/javascript;
+    gzip_proxied any;
+    gzip_min_length  1000;
+    gzip_disable "MSIE [1-6]\.";
```

Comment:
I've no idea if this is proper config, so I can't add much here

Reply:
It got rid of the lighthouse error so I assume it does.